.travis.yml

```diff
@@ -5,25 +5,28 @@ language: python
 python:
   - 3.5
   - 3.6
+  - 3.7
+  - 3.8-dev
+
+dist: xenial
+sudo: required
```

Comment:
This is irrelevant. It's a default now since Travis CI doesn't support container-based runners anymore.
```suggestion
```